.envrc

```diff
@@ -0,0 +1,30 @@
+# Nix config
+
+# if nix is installed, use it
+if [ ! -r .nix-disable  ] && has nix-env; then
+    export NIX_PROFILE=/nix/var/nix/profiles/eng-playbook
+    PATH_add ${NIX_PROFILE}/bin
+    nix_file=nix/default.nix
+
+    unset NIX_SSL_CERT_FILE
```

Comment:
Since most projects don't need this as noted in the How to, I wonder if we should comment it out instead of leave it in?

Reply:
Oh, yeah! done!